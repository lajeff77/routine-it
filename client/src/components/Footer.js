import React from 'react';
import AppBar from '@mui/material/AppBar';
import Toolbar from '@mui/material/Toolbar';
import Typography from '@mui/material/Typography';

const Footer = () => {
    return (
        <AppBar position="sticky" color="primary" sx={{alignItems: "center"}}>
            <Toolbar>
                <Typography variant="body1">Routine It &copy; 2024</Typography>
            </Toolbar>
        </AppBar>
    )
}

export default Footer;