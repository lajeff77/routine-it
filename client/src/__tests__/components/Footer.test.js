import { render } from "@testing-library/react";
import React from "react";
import Footer from "../../components/Common/Footer";

describe(Footer.name, ()=>{
    test('Footer matches snapshot', () => {
        const { container } = render(<Footer/>);
    
        expect(container).toMatchSnapshot();});
});