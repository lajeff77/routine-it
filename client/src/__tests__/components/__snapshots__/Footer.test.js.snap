// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      id="main-header"
    >
      <a
        class="header-logo"
        href="/"
      >
        <h1>
          Routine It
        </h1>
      </a>
      <a
        class="header-link"
        href="/"
      >
        <p>
          Home
        </p>
      </a>
      <a
        class="header-link"
        href="/signup"
      >
        <p>
          Sign Up
        </p>
      </a>
      <a
        class="header-link"
        href="/signin"
      >
        <p>
          Sign In
        </p>
      </a>
    </div>
    <div
      class="hero-section"
    >
      <h1>
        Everyone should be able to keep a routine!
      </h1>
      <a
        href="/signup"
      >
        <button>
          Get Started!
        </button>
      </a>
    </div>
    <div
      class="summary-section"
    >
      <h1>
        Traditional Routines Don't Work for Everyone.
      </h1>
      <p>
        As someone co-morbid with ADHD and Autism, I've never been able to keep a routine despite desperately needing one. I have a pattern. I see some motivational video or social media post about getting into a routine and getting my life on track. I've got boundless levels of energy and my meter is full. I think of one routine with a laundry list of tasks to do every single morning. I make it through the list for one day, two if I'm lucky. And then on the third day, I wake up with 50% energy in my meter. I maybe complete a few items, but I feel discouraged because I can't live up to my own expectations that I set when I had 100% energy. I'm disappointed in myself that I messed up so quick, and I abandon my list altogether.
      </p>
      <p>
        I've lived this cycle so many times, I ditched routines all together. I leaned on chaos and lack of structure, taking one day at a time. But I started feeling frustrated with that too. I lost velocity in my life, shifting wherever the wind (AKA my ADHD brain) takes every single day. Some things would go forgotten and get swept under the rug. My constant need for adjustment demolished my sense of direction. My autistic brain, who needs a clear path forward to achieve even the littlest of feats was lost.
      </p>
      <p>
        When you get lost, a compass can help guide you back to a path home (or maybe more realistically Google Maps in this day and age). I recently found my compass on YouTube, from a video titled 
        <a
          href="https://www.youtube.com/watch?v=JQFZNQS6eQ0&ab_channel=HayleyHoneyman"
        >
          Routines for ADHD that ACTUALLY WORK! üåõ‚òÄÔ∏è (Morning/Night routines)
        </a>
        . In this video, a girl named Haley who has an 
        <a
          href="https://www.neurodiversecouplescounseling.com/audhd"
        >
          AuDHD neurotype
        </a>
         like myself, explains her personal system (The Honeyman Method) for creating a routine that caters to both the variability of your energy when you have ADHD, and the need for structure when you have Autism. The system creates three tiers of your routine in order to allow yourself the grace to not stick to your most perfect ideal routine by having additional 2 scaled back routines for when you have an average energy day and a low energy day. I absolutely loved this idea and wanted to try it out thus Routine It was born, in order to organize and craft these routines.
      </p>
      <p>
        My vision for Routine It is to create an easy to use interface to allow myself and others to create tiered routine systems that work for them. Too many things in this world are made without neurodiverse people in mind. It's time for a routine planner for us by us.
      </p>
    </div>
    <header
      class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionSticky css-1fz1qzk-MuiPaper-root-MuiAppBar-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Routine It ¬© 2024
        </p>
      </div>
    </header>
  </div>
</DocumentFragment>
`;
